src/main/resources/com/google/api/codegen/js/sample.snip

```diff
@@ -0,0 +1,172 @@
+@snippet generate(class)
+  <!--
+  BEFORE RUNNING:
+  ---------------
+  1. If not already done, enable the {@class.apiTitle}
+     and check the quota for your project at
+     https://console.developers.google.com/apis/api/{@class.apiName}
+  2. Get access keys for your application. See
+     https://developers.google.com/api-client-library/javascript/start/start-js#get-access-keys-for-your-application
+  -->
+  <script src="https://apis.google.com/js/api.js"></script>
+  <script>
+  function {@class.makeApiCallFuncName}() {
+    @if class.needParams
+      var {@class.paramsVarName} = {
+        @join field : class.fields if class.fields on BREAK.add(BREAK)
+          @if field.description
+            @join line : util.getDocLines(field.description, 100)
+              // {@line}
+            @end
+
+
+          @end
+          {@field.name}: {@field.defaultValue},  // TODO: Update placeholder value.
```

Comment:
I believe trailing commas don't work prior to ES5 ... should we worry about this?